src/cli.rs

```diff
@@ -49,7 +49,11 @@ impl Volta {
     pub(crate) fn run(self, session: &mut Session) -> Fallible<ExitCode> {
         layout::ensure_volta_dirs_exist()?;
         if self.version {
-            println!("{}", env!("CARGO_PKG_VERSION"));
+            // suffix indicator for dev build
+            match cfg!(debug_assertions) {
+                true => println!("{}-dev", env!("CARGO_PKG_VERSION")),
+                false => println!("{}", env!("CARGO_PKG_VERSION")),
+            }
```

Comment:
While not a hard error, this is a warning in Clippy, which suggests using an `if` statement instead of matching on `true` and `false`, for readability:
```
if cfg!(debug_assertions) {
    println!("{}-dev", env!("CARGO_PKG_VERSION"));
} else {
    println!("{}", env!("CARGO_PKG_VERSION"));
```

Reply:
Yep that makes sense, thanks for the feedback, already updated in this PR.